vita3k/gui/src/gui.cpp

```diff
@@ -177,31 +182,26 @@ void init_icons(GuiState &gui, HostState &host) {
     }
 }
 
-void load_game_background(GuiState &gui, HostState &host, const std::string &title_id) {
+void load_game_background(GuiState &gui, HostState &host) {
     int32_t width = 0;
     int32_t height = 0;
     vfs::FileBuffer buffer;
 
-    vfs::read_app_file(buffer, host.pref_path, title_id, "sce_sys/pic0.png");
+    vfs::read_app_file(buffer, host.pref_path, host.io.title_id, "sce_sys/pic0.png");
     if (buffer.empty()) {
-        if (vfs::read_app_file(buffer, host.pref_path, title_id, "sce_sys/livearea/contents/template.xml")) {
-            LOG_INFO("Game background found in template for title {}.", title_id);
-            vfs::read_app_file(buffer, host.pref_path, title_id, "sce_sys/livearea/contents/bg.png");
-            vfs::read_app_file(buffer, host.pref_path, title_id, "sce_sys/livearea/contents/bg0.png");
-        } else {
-            if (gui.current_background != gui.user_backgrounds[host.cfg.background_image])
-                gui.current_background = gui.user_backgrounds[host.cfg.background_image];
-            LOG_WARN("Game background not found for title {}.", title_id);
-            return;
-        }
+        if (gui.current_background != gui.user_backgrounds[host.cfg.background_image])
```

Comment:
```suggestion
```

Reply:
No that is request

Reply:
LOL ok

Reply:
i same befàore, i want change only if is needed, master is already like that